web-client/src/app/components/header/header.component.ts

```diff
@@ -8,7 +10,15 @@ import { SessionQuery } from 'src/app/state/session.query';
 })
 export class HeaderComponent implements OnInit {
   @Input() title = 'NAUTILUS';
-  constructor(public sessionQuery: SessionQuery) {}
+  constructor(
+    public sessionQuery: SessionQuery,
+    private navController: NavController
+  ) {}
+
+  logoutAction() {
+    checkResetStores();
```

Comment:
I think we specifically want `resetStores` here, not `checkResetStores`: it's precisely when `persistAkitaState` is enabled that we really want this action to clear the session.